### ISSUE TITLE
End page for adventures

### ISSUE THREAD

**Issue by daytonellwanger** [daytonellwanger/ChooseYourAdventure#5]
When a user hits the end page of an adventure (i.e. a page with no actions), we need to render a "end of adventure" message below the page text and provide a link back to the home page.

In the future we'll also add a "give this adventure a rating" section.

**Comment by starfieldscreensaver** [comment]
Yeah, I was thinking of need for rating system earlier tonight.  Question, tiered like stars or just up or down?

**Comment by daytonellwanger** [comment]
Up to you!
